guides/common/modules/proc_customizing-the-discovery-pxe-boot.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Customizing_the_Discovery_PXE_Boot_{context}"]
+= Customizing the Discovery PXE boot
+[[Discovery_Templates_and_Snippets_Settings_{context}]]
+[[Provisioning_Template_PXELinux_Discovery_Snippet_{context}]]
+
+{Project} builds PXE boot menus from the global provisioning templates: `PXELinux global default` for BIOS provisioning, `PXEGrub global default`, and `PXEGrub2 global default` for UEFI provisioning.
+The PXE boot menus are available on {ProjectServer} and all {SmartProxies} that have the TFTP feature enabled.
+
+The Discovery menu item uses a Linux kernel for the operating system, which passes kernel parameters to configure the Discovery service.
+You can customize the passed kernel parameters by changing the following snippets:
+
+Snippet `pxelinux_discovery`:: This snippet is included in the `PXELinux global default` template.
++
+This snippet renders the Discovery boot menu option.
+The `KERNEL` and `APPEND` options boot the Discovery kernel and initial RAM disk.
+The `APPEND` option contains kernel parameters.
+
+Snippet `pxegrub_discovery`:: This snippet is included in the `PXEGrub global default` template.
+However, Discovery is http://projects.theforeman.org/issues/15997[not supported with GRUB 1.x].
+
+Snippet `pxegrub2_discovery`:: This snippet is included in the `PXEGrub2 global default` template.
++
+This snippet renders the Discovery GRUB2 menu entry.
+The `common` variable contains kernel parameters.
+
+To view provisioning templates and snippets, navigate to *Hosts* > *Templates* > *Provisioning Templates*.
```

Comment:
This should be the first step of your procedure. Otherwise, it looks odd if you tell the user to clone a snippet without explaining where they do this in the UI.